### PR TITLE
style(update text for SLL Tooltip)

### PR DESCRIPTION
**Pull request by gabester78**
### SUMMARY

Changed the SSL tooltip text to updated based on the database the user selects.
https://trello.com/c/B4i4wtpq/29-update-ssl-tooltip-text

### BEFORE/AFTER SCREENSHOTS OR ANIMATED GIF

Screen Shot 2021-09-22 at 12 49 32 PM

### TESTING INSTRUCTIONS

- From Welcome screen navigate to Data > Databases
- Click on "+ Database" button
- Click on PostgreSQL or MySql icons or select Amazon Redshift from the supported databases dropdown menu
- Hover over SLL Tooltip at the bottom of the form 

### ADDITIONAL INFORMATION

- [ ] Has associated issue:
- [ ] Required feature flags:
- [x] Changes UI
- [ ] Includes DB Migration (follow approval process in [SIP-59](https://github.com/apache/superset/issues/13351))
  - [ ] Migration is atomic, supports rollback & is backwards-compatible
  - [ ] Confirm DB migration upgrade and downgrade tested
  - [ ] Runtime estimates and downtime expectations provided
- [ ] Introduces new feature or API
- [ ] Removes existing feature or API

### CODE REVIEW

File: superset-frontend/src/views/CRUD/data/database/DatabaseModal/DatabaseConnectionForm.tsx [hunk 1]
```diff
@@ -437,7 +437,7 @@ const forceSSLField = ({
   changeMethods,
   db,
   sslForced,
-}: FieldPropTypes) => (
+}: FieldPropTypes) => { console.log("db", db); return (
```
Comment (lyndsiWilliams) [inline superset-frontend/src/views/CRUD/data/database/DatabaseModal/DatabaseConnectionForm.tsx:440]: Remove the console.log and remove the curly brackets here, we won't need any js logic here now that we're done debugging 😁

**Review by lyndsiWilliams — APPROVED** [review]
Just a missed console.log, this looks great!

File: superset-frontend/src/views/CRUD/data/database/DatabaseModal/DatabaseConnectionForm.tsx [hunk 2]
```diff
@@ -454,13 +454,30 @@ const forceSSLField = ({
...
+    {db?.database_name === 'Amazon Redshift' && (
+      <InfoTooltip
+        tooltip={t('Requires a root certificate authority (public, local, or self-signed). SSL Mode "verify-ca" will be used.')}
+        placement="right"
+        viewBox="0 -5 24 24"
+      /> )}
+
   </div>
-);
+  )
```
Comment (lyndsiWilliams) [inline superset-frontend/src/views/CRUD/data/database/DatabaseModal/DatabaseConnectionForm.tsx:480]: This parenthesis should have no tabbed space before it and a semicolon after it. Run the Prettify extension in VScode to catch these little things before GitHub CI does

**Review by lyndsiWilliams — APPROVED** [review]

File: superset-frontend/src/views/CRUD/data/database/DatabaseModal/DatabaseConnectionForm.tsx [hunk 3]
```diff
@@ -437,30 +437,45 @@ const forceSSLField = ({
...
+            tooltip={t('Requires a root certificate authority (public, local, or self-signed).')}
+            placement="right"
+            viewBox="0 -5 24 24" />)}
+
+        {db?.database_name === 'Amazon Redshift' && (
+          <InfoTooltip
+            tooltip={t('Requires a root certificate authority (public, local, or self-signed). SSL Mode "verify-ca" will be used.')}
+            placement="right"
+            viewBox="0 -5 24 24" />)}
+
```
Comment (eschutho) [inline superset-frontend/src/views/CRUD/data/database/DatabaseModal/DatabaseConnectionForm.tsx:475]: Thanks for this @gabester78! One suggestion that could make this database type agnostic and also clean up the three different InfoTooltips here would be to 
1) use an object for getting the different text, so something like: 

```
const tooltipText = {
    foo: "some text",
    bar: "other text",
}
```

and then 
```
<InfoTooltip
            tooltip={t(tooltipText(somevariable))}
            placement="right"
            viewBox="0 -5 24 24" />)}
```

Comment (eschutho) [reply superset-frontend/src/views/CRUD/data/database/DatabaseModal/DatabaseConnectionForm.tsx:475]: 2. you should be able to use the db.encryption_params.sslmode to switch between the different types of text, so you would pass the key in like `tooltip={t(tooltipText(db.encryption_params.sslmode))}`

Comment (gabester78) [reply superset-frontend/src/views/CRUD/data/database/DatabaseModal/DatabaseConnectionForm.tsx:475]: Great thanks! Should I go ahead and make the changes?

Comment (gabester78) [reply superset-frontend/src/views/CRUD/data/database/DatabaseModal/DatabaseConnectionForm.tsx]: Closing PR. Reference [](https://github.com/apache/superset/pull/16907) for updates.

**Comment by gabester78** [comment]
Continuing work at this [PR](https://github.com/apache/superset/pull/16907)

**Pull request closed without merging**